constants_local.xml

```diff
@@ -172,6 +172,7 @@
         <ENABLE_CHECK_PERFORMANCE_LOG>false</ENABLE_CHECK_PERFORMANCE_LOG>
         <FALLBACK_TEST_EPOCH>2</FALLBACK_TEST_EPOCH>
         <NUM_TXN_TO_SEND_PER_ACCOUNT>100</NUM_TXN_TO_SEND_PER_ACCOUNT>
+        <PACKET_EPOCH_LATE_ALLOW>1</PACKET_EPOCH_LATE_ALLOW>
```

Comment:
I think should move this to transactions section.  Tests section is just for internal testing stuff.

Reply:
I see